spec/support/helpers.rb

```diff
@@ -0,0 +1,44 @@
+module Helpers
```

Comment:
Can we have that by itself
`helper` just seems too generic and probably make this file grow to monster

![image](https://user-images.githubusercontent.com/470137/69717955-01be9f80-1105-11ea-98be-079420e679f3.png)

Reply:
Yep good idea